scripts/harvest_child.js

```diff
@@ -515,15 +515,16 @@ const main = async () => {
         stratsFiltered = stratsFiltered.concat(strats.filter(s => !s.shouldHarvest));
         stratsShouldHarvest = strats.filter(s => s.shouldHarvest);
 
-        // strats = stratsShouldHarvest.map(s => {
-        //   if (s.tvl < TVL_MINIMUM_TO_HARVEST) {
-        //     s.shouldHarvest = false;
-        //     s.notHarvestReason = `TVL is lower than min: ${TVL_MINIMUM_TO_HARVEST}`;
-        //   }
-        //   return s;
-        // });
-        // stratsFiltered = stratsFiltered.concat(strats.filter(s => !s.shouldHarvest));
-        // stratsShouldHarvest = strats.filter(s => s.shouldHarvest);
+				const i_TWO_WEEKS_AGO = new Date().getTime() / 1000 - 1209600;
```

Comment:
could you use camelCase for variables name?

Reply:
apply prettier (configuration is in .prettierrc file) to keep code clean in spaces